backend/module/chaos/serverexperimentation/xds/xds_cache.go

```diff
@@ -197,3 +228,13 @@ func getEnforcer(httpFaultConfig *serverexperimentation.HTTPFaultConfig) string
 		return "unknown"
 	}
 }
+
+func containsString(list []string, str string) bool {
+	for _, element := range list {
+		if element == str {
+			return true
+		}
+	}
+
+	return false
+}
```

Comment:
Instead of scanning each time I would build a `map[string]struct{}` and just do a map lookup